lib/utils/apiUtils.js

```diff
@@ -111,27 +109,27 @@ async function responseErrorHandler(
   callbackParameters
 ) {
   if (error && error.response) {
-    console.log(
+    consola.error(
       `Response Status: ${error.response.status} (${error.response.statusText}) - method: ${error.response.config.method} - url: ${error.response.config.url}`
     );
   }
   // Valid Request. Not Found
   if (error.response.status === 404) {
-    console.log(
+    consola.error(
       `Response Error (404): ${JSON.stringify(error.response.data.error)}`
     );
     return;
   }
   // Bad Request
   if (error.response.status === 400) {
-    console.log(
+    consola.error(
       `Response Error (400): ${JSON.stringify(error.response.data.error)}`
     );
     return;
   }
   // No access credentials
   if (error.response.status === 401) {
-    console.log(
+    consola.error(
```

Comment:
Also add this under `consola.debug` flag? It'll generally only trigger when refresh tokens are required. 

Unless the access token is completely invalid, we can have a separate error message to inform the user on that. 

But if code 401 + the error message is "The access token expired", we can skip any output in the console imo.

Reply:
Changed